kasa/iot/iotstrip.py

```diff
@@ -150,21 +162,37 @@ def on_since(self) -> datetime | None:
 
         return max(plug.on_since for plug in self.children if plug.on_since is not None)
 
-    async def current_consumption(self) -> float:
```

Comment:
I don't know why `current_consumption` is defined in as an async method as since 0.5 it seems to just get data from `realtime` state. This might be a breaking change but I think now is a better time to do it than after 0.7